test/_init_tests.js

```diff
@@ -160,6 +166,15 @@ class TestConfig {
     return this.if(this.platform.isIos.bind(this.platform));
   }
 
+  ifIe() {
+    // Necessary because we skip IE by default.
+    const ieSkipped = this.skipMatchers.indexOf(this.skipIe);
```

Comment:
Does this work? I think `skipMatchers` contains `this.platform.isIe.bind(this.platform)`, not `this.skipIe`. Probably want to save the former as an ivar and use it in both `skipIe()` and here.